sql/nm_spawn_points.sql

```diff
@@ -6976,6 +6976,7 @@ INSERT INTO `nm_spawn_points` VALUES (17203216,21,-156.857,0.458,485.462);
 INSERT INTO `nm_spawn_points` VALUES (17469747,0,169.270,8.028,-110.580);
 INSERT INTO `nm_spawn_points` VALUES (17469747,1,155.792,8.334,-124.494);
 INSERT INTO `nm_spawn_points` VALUES (17469747,2,169.525,11.964,-128.288);
+INSERT INTO `nm_spawn_points` VALUES (17818045,31,319.896,047.998,250.000);
```

Comment:
This mob already has this spawn point set in **mob_spawn_points.sql**, what is the need for this addition?

Reply:
I agree. This table is supposed to be for mobs that can spawn in a region randomly. Like KA, Behemoth, Fafnir, etc.
This mob is poped and its spawn point is the pop pos, right?